composer.json

```diff
@@ -14,32 +14,34 @@
         }
     ],
     "require": {
-        "php": ">=7.0.8",
+        "php": "^7.3|^8.0",
         "auto1-oss/service-api-components-bundle": "^1.0",
         "auto1-oss/service-api-request": "^1.0",
-        "symfony/serializer" : "~3.0|~4.0|~5.0",
-        "symfony/security-core" : "~3.0|~4.0|~5.0",
-        "symfony/security-csrf" : "~3.0|~4.0|~5.0",
-        "symfony/security-guard" : "~3.0|~4.0|~5.0",
-        "symfony/security-http" : "~3.0|~4.0|~5.0",
-        "symfony/monolog-bridge": "~3.0|~4.0|~5.0",
-        "symfony/dependency-injection": "~3.0|~4.0|~5.0",
-        "symfony/config": "~3.0|~4.0|~5.0",
-        "symfony/http-kernel": "~3.0|~4.0|~5.0",
-        "symfony/property-info": "^3.4.2|~4.0|~5.0",
-        "monolog/monolog": "~1.22",
+        "symfony/serializer" : "~3.0|~4.0|~5.0|~6.0",
+        "symfony/security-core" : "~3.0|~4.0|~5.0|~6.0",
+        "symfony/security-csrf" : "~3.0|~4.0|~5.0|~6.0",
+        "symfony/security-guard" : "~3.0|~4.0|~5.0|~6.0",
+        "symfony/security-http" : "~3.0|~4.0|~5.0|~6.0",
+        "symfony/monolog-bridge": "~3.0|~4.0|~5.0|~6.0",
+        "symfony/dependency-injection": "~3.0|~4.0|~5.0|~6.0",
+        "symfony/config": "~3.0|~4.0|~5.0|~6.0",
+        "symfony/http-kernel": "~3.0|~4.0|~5.0|~6.0",
+        "symfony/property-info": "^3.4.2|~4.0|~5.0|~6.0",
+        "monolog/monolog": "^1.22|^2.0|^3.0",
         "php-http/client-common": "^2.3",
         "php-http/discovery": "^1.3",
         "php-http/httplug": "^2.2",
         "psr/http-message": "^1.0",
-        "psr/log": "^1.1"
+        "psr/log": "^1.1|^2.0|^3.0",
+        "php-http/message-factory": "^1.1"
     },
     "require-dev": {
-        "symfony/yaml": "~3.0|~4.0|~5.0",
-        "symfony/console": "~3.0|~4.0|~5.0",
-        "phpunit/phpunit": "^4.8",
-        "symfony/phpunit-bridge": "^3.3.8",
-        "phpspec/prophecy": "^1.7.2"
+        "symfony/yaml": "~3.0|~4.0|~5.0|~6.0",
+        "symfony/console": "~3.0|~4.0|~5.0|~6.0",
+        "phpunit/phpunit": "^4.8|^8.5",
```

Comment:
Since 8.5 supports php 7.3 and this is only for development, I would drop the "support" for 4.8. The same for phpspec/prophecy-phpunit

Reply:
fixed, removed support + that package